app/views/development/index.html.erb

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>
+    government-frontend development page
+  </title>
+</head>
+<body>
+  <div id="wrapper">
+    <main class="govspeak">
+      <%= render 'govuk_component/title', title: 'government-frontend' %>
+
+      <% html = capture do %>
+        <p>
+          Hi, this is the root of the <%= link_to 'government-frontend', 'https://github.com/alphagov/government-frontend' %> application.
+          This page is intended to be shown in development and on Heroku review apps. It
+          won't be shown to users on GOV.UK because there are no routes to it.
+        </p>
+
+        <h2>Component guide</h2>
+
+        <p>
+          To see the available components, take a look at the <%= link_to 'component guide', '/component-guide' %>.
+        </p>
+
+        <h2>Configuration</h2>
+
+        <table>
+          <tr>
+            <td>Content store</td>
+            <td><%= link_to Plek.find('content-store'), Plek.find('content-store') %></td>
+          </tr>
+          <tr>
+            <td>Static</td>
+            <td><%= link_to Plek.find('static'), Plek.find('static') %></td>
+          </tr>
+          <tr>
+            <td>Rummager (search)</td>
+            <td><%= link_to Plek.find('rummager'), Plek.find('rummager') %></td>
+          </tr>
+        </table>
+
+        <h2>Examples</h2>
+
+        <p>
+          The content schemas contain <%= link_to 'handmade examples', 'https://github.com/alphagov/govuk-content-schemas/tree/master/examples' %>
```

Comment:
Could we also mention that we use these examples in our tests?

Reply:
I missed this bit, wonder if we can put it in a call out or make it more obvious the links below will not work

Reply:
I added a footnote to the links, hopefully clearer now.